django/core/management/color.py

```diff
@@ -4,18 +4,22 @@
 
 import functools
 import os
+import platform
 import sys
 
 from django.utils import termcolors
 
-try:
-    import colorama
+if platform.system() == "Windows":
```

Comment:
We normally use `sys.platform == "win32"`:
```suggestion
# Don't initialize colorama on non-Windows platforms.
if sys.platform == "win32":
```

Reply:
I will make the changes.